<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numkeyboard</class>
 <widget class="QWidget" name="numkeyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>numkeyboard</string>
  </property>
  <widget class="QLabel" name="backgroundNumKBLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="three_NumKBBut">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>190</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="two_NumKBBut">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>190</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="one_NumKBBut">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>190</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="six_NumKBBut">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>129</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="four_NumKBBut">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>129</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="five_NumKBBut">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>129</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nine_NumKBBut">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>69</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eight_NumKBBut">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>69</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seven_NumKBBut">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>69</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zero_NumKBBut">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>250</y>
     <width>116</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="period_NumKBBut">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>250</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minus_NumKBBut">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>69</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="devide_NumKBBut">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>8</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiple_NumKBBut">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>8</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Esc_NumKBBut">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>8</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_NumKBBut">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>129</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enter_NumKBBut">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>190</y>
     <width>52</width>
     <height>112</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>↵</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backspace_NumKBBut">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>8</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182,181,186);</string>
   </property>
   <property name="text">
    <string>←</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
